### DISCUSSION TITLE
Is there a complete example that demonstrates how to use stablehlo so that I can step through it （debug）.

### DISCUSSION THREAD

**Discussion by ckfgihub** [openxla/community#97] [stablehlo]
Is there a complete example that demonstrates how to use stablehlo so that I can step through it.

**Comment by ghpvnist** [comment]
We have a pending [PR](https://github.com/openxla/stablehlo/pull/1803/files#diff-0a35cfc54762787609d0cd7fae7ead1c300fde6dcaaf85051df4f9e131164b60) for documentation on StableHLO interpreter, and I've added a brief section on debugging. I hope this helps!
